lib/phoenix_live_view/test/live_view_test.ex

```diff
@@ -614,10 +627,28 @@ defmodule Phoenix.LiveViewTest do
       assert html =~ "The temp is: 30℉"
       assert render_submit(view, :refresh, %{deg: 32}) =~ "The temp is: 32℉"
   """
+  def render_submit(%Element{} = element, value, opts) do
```

Comment:
The submitter is only available when there is an element? If so... maybe we should support it on the `form` function instead? Sorry for the back and forth but the API is indeed tricky (on all cases!).

Reply:
Yes, we only do thorough form validation with an Element. Coupling submitter to `form()` prevents reusing the same form Element with multiple submitters, but that may be the least bad choice. No worries on the back and forth, I appreciate all the consideration!